semanticdb-java/src/main/protobuf/semanticdb.proto

```diff
@@ -103,12 +103,24 @@ message SymbolInformation {
   Kind kind = 3;
   int32 properties = 4;
   string display_name = 5;
+  repeated Annotation annotations = 13;
   Signature signature = 17;
   Access access = 18;
   repeated string overridden_symbols = 19;
   Documentation documentation = 20;
 }
 
+message Annotation {
+  Type tpe = 1;
+  // -- OUT OF SPEC -- //
+  message ParameterPair {
```

Comment:
I think `AssignTree` makes sense here. A scala indexer could emit the `Tree` they'd emit for the `value` field instead of adding the `AssignTree` layer as they dont have to name the parameters, while the java indexer would use the `AssignTree`, so that seems like a good solution to me :slightly_smiling_face:

Reply:
How generic do we want to make `AssignTree`? LHS could be more than an `IdTree`, could be a `SelectTree` or an array access tree 

https://docs.oracle.com/javase/specs/jls/se8/html/jls-15.html#jls-15.26

Reply:
Good point! Let's make lhs `Tree` in that case 👍

Reply:
The Scalameta `Assign` node allows trees in both RHS and LHS  https://github.com/scalameta/scalameta/blob/f39cbb3e70b627afc72c16fc99990f2c6bcffa9b/scalameta/trees/shared/src/main/scala/scala/meta/Trees.scala#L95